Screensharing/Agora-Screen-Sharing-Android/screensharemodule/src/test/java/com/example/screensharemodule/ExampleUnitTest.java

```diff
@@ -0,0 +1,17 @@
+package com.example.screensharemodule;
```

Comment:
do not use com.sample for name package.

Maybe io.agora.ext.video/ScreenCaptureSource?